src/Symfony/Component/HttpKernel/Exception/DisableLogDirectoryException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Exception;
+
+use Symfony\Component\DependencyInjection\Exception\RuntimeException;
+
+/**
+ * DisableLogDirectoryException.
+ *
+ * @author Asmir Mustafic <goetas@gmail.com>
+ */
+class DisableLogDirectoryException extends RuntimeException
```

Comment:
Should we find a more generic name here, such as MethodNotImplementedException (same name in other components)